src/pages/AddProject.tsx

```diff
@@ -66,6 +74,25 @@ class AddProject extends React.Component<RouteComponentProps, FormState> {
     cancelModalOpen: false
   }
 
+  lookingForOptions: FormOptions = [
```

Comment:
Let's add a comment at the top that these values will be coming in from a getPosition API when #114 is implemented.